.github/workflows/npm-release-publish.yml

```diff
@@ -32,6 +35,12 @@ jobs:
       - name: Install dependencies
         run: rush install
 
+      # Switch flavor to stable when it is release branch
+      - name: Switch flavor for stable build
+        # If no param for release flavor, get it from branch name
+        if: ${{ !github.event.inputs.branch && contains(github.ref, 'beta') || github.event.inputs.branch === 'stable' }}
```

Comment:
should this be `github.event.inputs.release-flavor` instead of `github.event.inputs.branch`?

Reply:
and should it be `contains(github.ref, 'stable')` instead of beta?

Reply:
Addressed the first comment - for second one, we don't have str like 'stable' in stable branch, besides, it is also recommended to use the parameter instead of leaving it empty, so this is more like a fallback

Reply:
Does 
![image](https://user-images.githubusercontent.com/2684369/154763461-9fdadedd-241e-499e-a121-15a4ad9fc457.png)
not need updated to release-flavor also?

Reply:
Does this say, if github.ref contains beta then do a rush-switch:stable? That seems odd

Reply:
Should it not be `!contains(github.ref, 'beta')`?

Reply:
Ohhh! That is correct, it is so sad that I can't test this myself easily, thanks for catching this